### DISCUSSION TITLE
how do i make the whole status bar refresh every second

### DISCUSSION THREAD

**Discussion by Prajwalg19** [tobi-wan-kenobi/bumblebee-status#939] [Q&A]
the status bar configured with bumblebee refreshes every 6 seconds in my case
your doc page suggests a technique to change the interval of particular modules but when I follow the doc it refreshes every 6+1second instead of refreshing every 1 second. This causes the seconds part in the time module to refresh every 6 seconds This is the link for the bumblebee installation
I'm expecting a solution for refreshing my status bar every 1 second because seconds part of the time module refreshes every 6 seconds.
While the laptop is charging it refreshes every second but when i unplug it it refreshes every 6 seconds

**Comment by tobi-wan-kenobi** [comment]
Can you please show your config (modules, parameters, etc.)?

Any steps on how to reproduce this behaviour would be much appreciated.

**Reply by Prajwalg19** [reply]
[config.zip](https://github.com/tobi-wan-kenobi/bumblebee-status/files/9889357/config.zip)

i have my bumblebee cloned file in Downloads directory. all i did was specify path to the python executable inside the bar block of my i3config. while charging the status bar works alright , i can increase the volume by scrolling mouse wheel , seconds part in time module increments every second, but when unpluged seconds increment every 6 seconds and volume increments or decrements on every refresh that takes 6 seconds.

**Reply by Prajwalg19** [reply]
it works fine while charging ,so something related to it should be the solution but i dont know how to figure it out because i'm weak at python

**Reply by tobi-wan-kenobi** [reply]
I think this is likely not related to Python at all, but rather tocpowet management. Is your system generally noticably slower when on battery?

**Reply by Prajwalg19** [reply]
nothing as such. I'm running this on ubuntu 22.04 LTS.
